crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -20,7 +20,7 @@ use bevy_render::{
     },
     prelude::Color,
     render_graph::{NodeRunError, RenderGraphApp, RenderGraphContext, ViewNode, ViewNodeRunner},
-    render_resource::*,
+    render_resource::{BindGroupEntries, *},
```

Comment:
is `BindGroupEntries` not part of `*`?

Reply:
guess i got a little overeager with my ctrl+enter fingers